azurerm/resource_arm_analysis_services_server_test.go

```diff
@@ -0,0 +1,379 @@
+//	resourceName := "azurerm_analysis_services_server.test"
+//	ri := tf.AccRandTimeInt()
+//	preAdminUsers := []string{"admin@domain.tld"}
+//	postAdminUsers := []string{"admin@domain.tld", "admin2@domain.tld"}
+//	preConfig := testAccAzureRMAnalysisServicesServer_adminUsers(ri, testLocation(), preAdminUsers)
+//	postConfig := testAccAzureRMAnalysisServicesServer_adminUsers(ri, testLocation(), postAdminUsers)
+//
+//	resource.ParallelTest(t, resource.TestCase{
+//		PreCheck:     func() { testAccPreCheck(t) },
+//		Providers:    testAccProviders,
+//		CheckDestroy: testCheckAzureRMAnalysisServicesServerDestroy,
+//		Steps: []resource.TestStep{
+//			{
+//				Config: preConfig,
+//				Check: resource.ComposeTestCheckFunc(
+//					testCheckAzureRMAnalysisServicesServerExists(resourceName),
+//					resource.TestCheckResourceAttr(resourceName, "admin_users.#", "1"),
+//					resource.TestCheckResourceAttr(resourceName, "admin_users.0", preAdminUsers[0]),
+//				),
+//			},
+//			{
+//				Config: postConfig,
+//				Check: resource.ComposeTestCheckFunc(
+//					testCheckAzureRMAnalysisServicesServerExists(resourceName),
+//					resource.TestCheckResourceAttr(resourceName, "admin_users.#", "2"),
+//					resource.TestCheckResourceAttr(resourceName, "admin_users.0", postAdminUsers[0]),
+//					resource.TestCheckResourceAttr(resourceName, "admin_users.1", postAdminUsers[1]),
+//				),
+//			},
+//		},
+//	})
+//}
+
+func testAccAzureRMAnalysisServicesServer_basic(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_analysis_services_server" "test" {
+  name                = "acctestass%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku 				  = "B1"
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMAnalysisServicesServer_withTags(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_analysis_services_server" "test" {
+  name                = "acctestass%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku 				  = "B1"
+
+  tags = {
+	label = "test"
+  }
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMAnalysisServicesServer_withTagsUpdate(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_analysis_services_server" "test" {
+  name                = "acctestass%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku 				  = "B1"
+
+  tags = {
+	label = "test1"
+	env   = "prod"
```

Comment:
Could we ensure capitals work here?
```suggestion
	ENV   = "Prod"
```

Reply:
changed to uppercase